src/value-editor/meta-editors/object/object.value-editor.tpl.pug

```diff
@@ -26,6 +26,7 @@ fieldset.form-horizontal("{{withoutNgForm ? '' : 'ng-form'}}"="{{withoutNgForm ?
                 type="transformedFieldEditor.type"
                 is-disabled="transformedFieldEditor.isDisabled || $ctrl.valueEditorController.isDisabled"
                 is-visible="$ctrl.trueIfUndefined($ctrl.valueEditorController.isVisible)"
+                is-focused="$ctrl.valueEditorController.isFocused && $first"
```

Comment:
Tady si myslim, ze by se to melo chovat jinak. Pokud to chapu spravne, v tomto pripade to je tak, ze se vzdycky zafocusuje prvni value editor v object editoru. Co kdyz ale budu chtit zafocusovat treba treti editor? Vytvorim JSON definici s tim, ze tretimu editoru dam `isFocused: true`,ale stejne mi to zafocusuje jen ten prvni. Zde bych to tedy udelal tak, ze kdyz nebude zadny vnitrni editor zafocusovany, ale object editor bude mit `isFocused: true`, zafocusuju prvni element. Pokud nebude, nebudu focusovat nic.